CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 BUG FIXES:
  * core: Fix search endpoint forwarding for multi-region clusters [[GH-3680](https://github.com/hashicorp/nomad/issues/3680)]
  * config: Revert minimum CPU limit back to 20 from 100.
+ * discovery: Fix handling of `service.check_restart` [[GH-3718](https://github.com/hashicorp/nomad/issues/3718)]
```

Comment:
Allow `check_restart` to be specified in the `service` stanza.

Reply:
Move to Feature or leave as Bug Fix?

I'm going to move to Feature since we removed it from the docs and it never worked but let me know if Bug Fix is preferred.